packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1866,7 +1887,212 @@ export class MergeTree {
 		}
 	}
 
-	public obliterateRange(
+	private obliterateRangeSided(
```

Comment:
this adds about 3kb to the bundle size? can we reduce the impact by sharing code?